creator.pro

```diff
@@ -161,7 +161,7 @@ linux* {
     # manually add suffix 32/64
     TARGET = LibreELEC.USB-SD.Creator.Linux-bit.bin
 
-    QMAKE_CXXFLAGS += -std=c++11
+    QMAKE_CXXFLAGS += -std=c++17

```

Comment:
1. C++ standard can be specified in `CONFIG`
2. I think it makes sense to use the same for all platforms, something like "if Qt >= 6 then C++17 else C++11"

Reply:
I'm confident we are never going to build with anything older than Qt6 now so that might not be important :)